spec/features/application/admin_can_approve_pet_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "rails_helper"
 
 RSpec.describe "approve pet", type: :feature do
-  before :all do
+  before :each do
```

Comment:
before :all is why the tests were failing. I will be extra careful about using this in the future!